### ISSUE TITLE
[ENH] - Re-add the blue "+" button in the top left corner to open the launcher.

### ISSUE THREAD

**Issue by Adam-D-Lewis** [nebari-dev/jupyterlab-new-launcher#51]
### Feature description

I miss the  blue "+" button in the top left corner to open the launcher.  Now I have to click File/New to open the launcher.  Is it possible to readd the blue "+" to open the launcher?  I don't think there's another way to do so, right?

### Value and/or benefit

Familiar UX

**Comment by krassowski** [comment]
I also noticed this recently, totally agree. This looks like a bug as the code for activating the button is included in this extension:

https://github.com/nebari-dev/jupyterlab-new-launcher/blob/5aafb2ac13f17764778e72d3d3e60ca04bc5c3b8/src/index.ts#L202-L212

but for some reason it does not make the button appear :thinking:

**Comment by krassowski** [comment]
Nah, this is the other add button. The one we are missing would need to be added by schema like in https://github.com/jupyterlab/jupyterlab/blob/130089eddb330a44f7f9d496b1e64eddcb497d6b/packages/launcher-extension/schema/plugin.json
